backend/src/lib.rs

```diff
@@ -65,6 +66,11 @@ pub fn setup_actix_app(
         .service(scope("/tasks").configure(routes::task::config))
         .service(scope("/user").configure(routes::user::config))
         .service(scope("/event").configure(routes::event::config))
+        .service(
```

Comment:
I think you can merge that with /user and in future we can rename it to /account